src/main/config/config_master.h

```diff
@@ -24,7 +24,8 @@ typedef struct master_t {
     uint8_t magic_be;                       // magic number, should be 0xBE
 
     uint8_t mixerMode;
-    uint32_t enabledFeatures;
+    uint32_t enabledFeatures;               // features that are currently enabled (working set)
+    uint32_t desiredFeatures;               // features to be enabled after reboot (feature latch)
```

Comment:
IIRC It should be posible to use the backup ram.  Look at the systemReset() in system_stm32f10x.c.  Be aware of #619 and #620